src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/08.md

```diff
@@ -3,48 +3,48 @@
 ##### Historical period: the 16th-17th centuries CE
 ##### Scene of action: Bavaria
 
-Right after the beer trade had begun, early laws regulating it emerged. Two main issues are bothering authorities of all the brewing regions: ensuring the quality of the product and taxation. It's quite possible to take some specific areas and observe how the development of the brewing industry leads to the local legislation becoming more and more strict[ref:unger-middle-ages:69]().
+Right after the beer trade had begun, early laws emerged for its regulation. Two main issues had vexxed authorities of all the brewing regions: ensuring the quality of the product and taxation. It's quite possible to track these specific aspects and to observe how the development of the brewing industry led to increasingly rigid and complex local legislation[ref:unger-middle-ages:69]().
```

Comment:
@JoRembert I've replaced the first 'observe' with 'track', otherwise it's duplicate 'observes'

Reply:
yes!